packages/ui-view/src/View/styles.ts

```diff
@@ -243,47 +192,6 @@ const getFocusStyles = (props: ViewProps, componentTheme: ViewTheme) => {
     success: componentTheme.focusColorSuccess,
     danger: componentTheme.focusColorDanger
   }
-
-  const focusRingRadius = getFocusRingRadius(borderRadius, props)
-  const focusRingVariants: PartialRecord<FocusRingRadius, string | 0> = {
-    'focusRing--radiusInherit': 'inherit',
-    'focusRing--radiusNone': 0
-  }
-  const borderRadiusByOffset: {
-    [k in NonNullable<ViewProps['focusPosition']>]: PartialRecord<
-      FocusRingRadius,
-      { borderRadius: string }
-    >
-  } = {
-    offset: {
-      'focusRing--radiusSmall': {
-        borderRadius: `calc(${componentTheme.borderRadiusSmall} + (${componentTheme.focusOutlineOffset} - ${componentTheme.focusOutlineWidth}))`
-      },
-      'focusRing--radiusMedium': {
-        borderRadius: `calc(${componentTheme.borderRadiusMedium} + (${componentTheme.focusOutlineOffset} - ${componentTheme.focusOutlineWidth}))`
-      },
-      'focusRing--radiusLarge': {
-        borderRadius: `calc(${componentTheme.borderRadiusLarge} + (${componentTheme.focusOutlineOffset} -  ${componentTheme.focusOutlineWidth}))`
-      },
-      'focusRing--radiusCustom': {
-        borderRadius: `calc(${focusRingBorderRadius} + (${componentTheme.focusOutlineOffset} - ${componentTheme.focusOutlineWidth}))`
-      }
-    },
-    inset: {
-      'focusRing--radiusSmall': {
-        borderRadius: `calc(${componentTheme.borderRadiusSmall} - (${componentTheme.focusOutlineInset} + ${componentTheme.focusOutlineWidth}))`
-      },
-      'focusRing--radiusMedium': {
-        borderRadius: `calc(${componentTheme.borderRadiusMedium} - (${componentTheme.focusOutlineInset} + ${componentTheme.focusOutlineWidth}))`
-      },
-      'focusRing--radiusLarge': {
-        borderRadius: `calc(${componentTheme.borderRadiusLarge} - (${componentTheme.focusOutlineInset} + ${componentTheme.focusOutlineWidth}))`
-      },
-      'focusRing--radiusCustom': {
-        borderRadius: `calc(${focusRingBorderRadius} - (${componentTheme.focusOutlineOffset} + ${componentTheme.focusOutlineWidth}))`
-      }
-    }
-  }
```

Comment:
the cool part is that we're getting rid of this spaghetti code. I think it was needed because it needed to 'copy' CSS props to the `:before` tag previously